From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Thu, 14 Mar 2024 23:24:06 +0100
Subject: [PATCH] info-overview: Install bare logos into pixmaps, not icons

We prefer having images that are not part of icon themes in the pixmaps
directory.
---
 panels/info-overview/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/info-overview/meson.build b/panels/info-overview/meson.build
index af5213bf3ade..899e77b7961e 100644
--- a/panels/info-overview/meson.build
+++ b/panels/info-overview/meson.build
@@ -69,7 +69,7 @@ executable(
 
 install_data(
   [ 'gnome-logo-text.svg', 'gnome-logo-text-dark.svg' ],
-  install_dir: join_paths(control_center_datadir, 'icons')
+  install_dir: join_paths(control_center_datadir, 'pixmaps')
 )
 
 subdir('icons')
